test/qa/lib/server.py

```diff
@@ -50,3 +52,17 @@ def get_hostname_by(technology="", protocol="", obfuscated="", group_id=""):
     available_server_count = len(requests.get(url).json()) - 1
     server = requests.get(url).json()[random.randint(0, available_server_count)]
     return server["name"], server["hostname"]
+
+
+def get_city_name(server_name):
```

Comment:
Wouldn't it be better to combine the 2 functions and return some kind of class, e.g. `get_server_info(server)`? In this way the information will be obtained from one server call + if more data is needed in the future, then more members are added to the class?